CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this
+repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your
+interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the
+   layer when doing a build.
+2. Update the `README.md` and `CHANGELOG.md` with details of changes to the
+   interface, this includes new environment variables, exposed ports, useful
+   file locations and container parameters.
```

Comment:
Some of this reads like boiler plate and is not relevant to OnDemand right now. For example:

>exposed ports, container parameters